### DISCUSSION TITLE
Account deletion requirements

### DISCUSSION THREAD

**Discussion by Gillissie** [lupidan/apple-signin-unity#150] [Q&A]
In reference to...
https://developer.apple.com/news/?id=12m75xbj#:~:text=As%20a%20reminder%2C%20apps%20that,1(v).

Apple says "If your app offers Sign in with Apple, you’ll need to use the Sign in with Apple REST API to revoke user tokens when deleting an account."

Is this something you plan on adding to this plugin? I mean revoking user tokens.

**Comment by lupidan** [comment]
No.

The REST API should be something you use from your backend, not from the client.

![Screenshot_20220630_081352.jpg](https://user-images.githubusercontent.com/1314521/176605684-91e88e37-9e66-4e03-893e-0d7f20d556bf.jpg)

**Reply by Gillissie** [reply]
Ok, that makes sense. Does this mean we're supposed to store the user token that is given when the user signs in, so we can revoke it later? If so, can you confirm how we get that token?

**Reply by lupidan** [reply]
> Does this mean we're supposed to store the user token that is given when the user signs in, so we can revoke it later?

Well, yeah...

> can you confirm how we get that token?

All tokens are obtained as a IAppleIdCredential after the user performs a SIWA attempt. It's your obligation to send that info to your backend for that particular user.
